modules/virt-using-disk-image-install-windows.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
+[id="virt-using-disk-image-install-windows_{context}"]
+= Using a disk image to install Windows
+
+Microsoft does not provide disk images for download, but you can create a disk image using the Windows DVD. This disk image can then be used to create virtual machines.
+
+.Procedure
+
+. Use the {VirtProductName} web console VM wizard to xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[create a new Windows VM], using the template for your desired Windows version.
+. Select the DVD image as the boot source.
+. Uncheck *Clone available operating system source to this Virtual Machine*.
+. Clear the *Start this virtual machine after creation* checkbox.
+. Click *Customize virtual machine* -> *Advanced*
+. Under *Sysprep*, specify the `autounattend.xml` answer file settings following link:https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/update-windows-settings-and-scripts-create-your-own-answer-file-sxs[Microsoft guidelines].
+. In the YAML, replace `running:false` with `runStrategy: RerunOnFailure`, and save. The VM will start automatically.
+. The `sysprep` disk containing the autounattend.xml answer file is now attached to the VM.
```

Comment:
This is not a new step. This should be attached to the previous step as the result. Also, I think autounattend.xml needs to be in backticks because it is the name of an answer file.